backend/tests/v5/helper/services.js

```diff
@@ -68,31 +68,28 @@ db.createModel = (teamspace, _id, name, props) => {
 	return DbHandler.insertOne(teamspace, 'settings', settings);
 };
 
-db.createRevision = (teamspace, modelId, { _id, author, timestamp, tag }) => {
-	const rev = {
-		_id: stringToUUID(_id),
-		author,
-		timestamp,
-		tag,
-	};
-	return DbHandler.insertOne(teamspace, `${modelId}.history`, rev);
+db.createRevision = (teamspace, modelId, revision) => {
+	const formattedRevision = { ...revision, _id: stringToUUID(revision._id) };
+	return DbHandler.insertOne(teamspace, `${modelId}.history`, formattedRevision);
 };
 
 ServiceHelper.generateUUIDString = () => uuidToString(generateUUID());
 ServiceHelper.generateRandomString = () => Crypto.randomBytes(15).toString('hex');
+ServiceHelper.generateRandomDate = (start, end) => new Date(start.getTime() + Math.random()
```

Comment:
Might be good to give it default params so whoever use it don't have to think about it
```suggestion
ServiceHelper.generateRandomDate = (start = new Date(2018, 1, 1), end = new Date()) => new Date(start.getTime() + Math.random()
```